rules/generate-insightful-reports-via-company-emails/rule.md

```diff
@@ -0,0 +1,24 @@
+---
+type: rule
+title: Do you generate insightful reports via company emails?
+uri: generate-insightful-reports-via-company-emails
+authors:
+  - title: Isabel Sandstroem
+    url: https://www.ssw.com.au/people/isabel-sandstroem/
+created: 2023-12-15T01:24:13.247Z
+guid: 9477fb4c-b872-4c82-af96-34d1cabd1d7c
+---
+The emails exchanged within your company are more than just communication tools; they are a rich source of data. By analyzing this data, we can gain insights into various aspects of our business operations.
+
+<!--endintro-->
+
+At SSW, we follow a set of [email rules](www.ssw.com.au/rules/rules-to-better-email/) that ensure our emails are consistent and structured. This consistency is crucial for effective data analysis, as it standardizes the format and content of the emails, making them easier to parse and understand.  
```

Comment:
```suggestion
Emails should follow a set of [email rules](www.ssw.com.au/rules/rules-to-better-email/) that make communication consistent and structured. When emails have standard format and content, data analysis is possible because the content can be parsed.  
```